onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -87,7 +87,8 @@ class ConvActivationSelector : public NodeSelector {
       if (graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Relu", {6, 13, 14}) ||
           graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Sigmoid", {6, 13}) ||
           graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "Tanh", {6, 13}) ||
-          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "LeakyRelu", {6, 16})) {
+          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "LeakyRelu", {6, 16}) ||
+          graph_utils::IsSupportedOptypeVersionAndDomain(activation_node, "HardSigmoid", {6})) {
```

Comment:
This change is in common part. Not sure whether I should limit it only for jsep. But I think it should work for other eps since `HardSigmoid` is a unary op.

cc @fs-eire @skottmckay

Reply:
@pranavsharma

Reply:
see this part of the code:
https://github.com/microsoft/onnxruntime/blob/0c2f0ba90da11ad53c63810e5f3e6fda4e295899/onnxruntime/core/optimizer/conv_activation_fusion.cc#L114-L118

I'm not sure if it works for all other EPs, but at least the CPU EP also handles it. would it be safer to add the JSEP to this condition? in any event, I think we don't need "HardSigmoid" handling in two places.

FYI, this comment seems to be the original motivation for limiting support to the CPU EP.
https://github.com/microsoft/onnxruntime/pull/8674#discussion_r686239781

Reply:
To be safe I would limit this to JSEP only. This particular fusion has had its share of problems in the past.

Reply:
Done. Move it together with CPU EP. Thanks for your suggestion.